### DISCUSSION TITLE
Is it possible to share a global between windows and dialogs?

### DISCUSSION THREAD

**Discussion by GavinShox** [slint-ui/slint#6750] [Q&A]
For some reason I cannot share a global singleton between a dialog and a window. State doesn't seem to be shared e.g. a two way binding between a global property and a dialog property isn't updating the global when read from the Window component. Is this some limitation I'm not aware of? Can't find any documentation that explains this. Thanks!

**Comment by TheColorRed** [comment]
Do you have some example code?
